### PR TITLE
fix: Use standard macros for debug message formating

### PR DESCRIPTION
**Pull request by ylm**

### CODE REVIEW

**Comment by dragonmux** [comment]
Thank you for the contribution! Unfortunately, this is a duplicate of the ADIv5 cleanup commit in #1139 - certainly can't blame you for not knowing that though.

For reasons to do with the way the constant is presently defined, formatting it actually has to be done with `"%lX"` rather than `"%" PRIx32`- If you alter this PR to make the definition of the constant use the `UINT32_C` macro to eliminate this problem, we'll take this change over our own.

**Comment by ylm** [comment]
Would it make sense for me to cherry-pick your ADIv5 commit and include it as part of this PR for thoroughness as it's not a complete overlap?

**Comment by dragonmux** [comment]
yes, that does make sense.

**Comment by ylm** [comment]
That should do it

**Comment by ylm** [comment]
Thanks a lot for the code review

File: src/target/adiv5.c [hunk 1]
```diff
@@ -64,9 +64,9 @@
  *                                 V
  *                           Component ID
  */
-#define CID_PREAMBLE    0xB105000DUL
-#define CID_CLASS_MASK  0x0000F000UL
-#define CID_CLASS_SHIFT 12UL
+#define CID_PREAMBLE    UINT32_C(0xB105000D)
+#define CID_CLASS_MASK  UINT32_C(0x0000F000)
+#define CID_CLASS_SHIFT UINT32_C(12)
```
Comment (dragonmux) [inline src/target/adiv5.c:69]: This one can and should just be `12U`

**Review by dragonmux — APPROVED** [review]
LGTM, there's one advisory item but this is otherwise ready for merge

**Review by dragonmux — APPROVED** [review]
LGTM, merging.

**Pull request merged**